Rocket.Chat/Controllers/Chat/ChatControllerUploader.swift

```diff
@@ -261,4 +261,27 @@ extension ChatViewController {
         }
     }
 
+    func uploadDialog(_ file: FileUpload) {
+        let alert = UIAlertController(title: localized("alert.upload_dialog.title"), message: "", preferredStyle: .alert)
+        alert.addTextField { (_ fileName) -> Void in
+            fileName.placeholder = localized("alert.upload_dialog.placeholder.title")
+            fileName.text = file.name
+        }
+        alert.addTextField { (_ fileDescription) -> Void in
+            fileDescription.placeholder = localized("alert.upload_dialog.placeholder.description")
+        }
+        alert.addAction(UIAlertAction(title: localized("alert.upload_dialog.action.upload"), style: .default, handler: { _ in
+            let name = alert.textFields?[0].text ?? ""
```

Comment:
Just one idea, to prevent crashing here... what about saving the textField instance in a method variable and then using it inside of the block? Like this...

```swift
var textFieldName: UITextField?

alert.addTextField { (_ fileName) -> Void in
   textFieldName = fileName
   ...
}
...

alert.addAction(UIAlertAction(title: localized("alert.upload_dialog.action.upload"), style: .default, handler: { _ in
    let name = textFieldName?.text ?? ""
    ...
}))
```